app/main/controllers/template/rtmedia-functions.php

```diff
@@ -534,7 +534,13 @@ function rtmedia_media( $size_flag = true, $echo = true, $media_size = 'rt_media
 	if ( isset( $rtmedia_media->media_type ) ) {
 		if ( 'photo' === $rtmedia_media->media_type ) {
 			$src  = wp_get_attachment_image_src( $rtmedia_media->media_id, $media_size );
-			$html = "<img src='" . esc_url( $src[0] ) . "' alt='" . esc_attr( $rtmedia_media->post_name ) . "' />";
+
+			/**
+			 * Used `set_url_scheme` because it breaks the image if there is special characters are there into image name.
```

Comment:
did you mean? 
Used `set_url_scheme` because `esc_url` breaks the image if there is special characters are there into image name.